### DISCUSSION TITLE
Bootstrap 5 Javascript

### DISCUSSION THREAD

**Discussion by oliverw** [Megabit/Blazorise#5108] [Q&A] [answered]
Does Blazorise implement its Bootstrap 5 components using JSInterop to `bootstrap.js` or why is the inclusion of Bootstrap's own Javascript required?

**Comment by David-Moreira** [comment, marked as answer]
Hello @oliverw 

Please find the following comment on the bs5 usage docs:
```
 <!-- These are the standard js dependencies this provider tipically dependes upon, but Blazorise deems these as optional as Blazorise Components should work correctly without these  -->
  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.1/dist/js/bootstrap.bundle.min.js" integrity="sha384-/bQdsTh/da6pkI1MST/rWKFNjaCP5gBSY4sEBT38Q/9RBh9AH40zEOg7Hlq2THRZ" crossorigin="anonymous"></script>
```

https://blazorise.com/docs/usage/bootstrap5

Unless you find something not working correctly, UI wise, it should be safe to not use the bs5 js at all. In fact our docs website that's in bs5 does not use it.

**Reply by oliverw** [reply]
Thanks for the clarification.
